pyTigerGraph/gds/gds.py

```diff
@@ -819,6 +819,147 @@ def edgeNeighborLoader(
         else:
             return EdgeNeighborLoader(**params)
 
+    def nodepieceLoader(self, 
+                        v_feats: Union[list, dict] = None,
+                        target_vertex_types: Union[str, list] = None,
+                        compute_anchors: bool = False,
+                        use_cache: bool = False,
+                        clear_cache: bool = False,
+                        anchor_method: str = "random",
+                        anchor_cache_attr: str = "anchors",
+                        max_distance: int = 5,
+                        max_anchors: int = 10,
+                        max_relational_context: int = 10,
+                        anchor_percentage: float = 0.01,
+                        anchor_attribute: str = "is_anchor",
+                        e_types: list = None,
+                        global_schema_change: bool = False,
+                        tokenMap: Union[dict, str] = None,
+                        batch_size: int = None,
+                        num_batches: int = 1,
+                        shuffle: bool = False,
+                        filter_by: str = None,
+                        loader_id: str = None,
+                        buffer_size: int = 4,
+                        reverse_edge: bool = False,
+                        timeout: int = 300000) -> NodePieceLoader:
+        """Returns a `NodePieceLoader`instance.
+        A `NodePieceLoader` instance loads all edges from the graph in batches, along with the vertices that are connected with each edge.
+
+        The NodePiece algorithm borrows the idea of "tokenization" from Natural Language Processing. The dataloader offers the functionality
+        to "tokenize" the graph in the form of randomly selecting "anchor vertices". If you are running NodePiece for the first time,
+        anchors have to be created.
+
+        NOTE: When you initialize the loader on a graph for the first time,
+        the initialization might take a minute as it installs the corresponding
```

Comment:
Rather than "might take a minute" I suggest:

NOTE: The first time you initialize the loader on a graph, it must first install the corresponding query to the database. However, the query installation only needs to be done once, so you will not need to wait when you initialize the loader on the same graph again.